conctr.agent.class.nut

```diff
@@ -10,8 +10,13 @@ class Conctr {
     static version = [1,0,0];
 
     static DATA_EVENT = "conctr_data";
+    static LOCATION_REQ = "conctr_get_location";
+    static LOCATION_RESP = "conctr_location";
+    static AGENT_OPTS = "conctr_agent_options";
+    static SOURCE_DEVICE = "impdevice";
```

Comment:
From the devices sendData function I know it could only have been called from the device so i can set the _source to impdevice. In the agent sendData is called either by the device or directly through the agent code and rather than force the user to put in a _source, i thought this was a better way to do it?